scm/driver/github/repo.go

```diff
@@ -110,6 +114,14 @@ func (s *RepositoryService) List(ctx context.Context, opts scm.ListOptions) ([]*
 	return convertRepositoryList(out), res, err
 }
 
+// ListV2 returns the user repository list based on the searchTerm passed.
+func (s *RepositoryService) ListV2(ctx context.Context, opts scm.RepoListOptions) ([]*scm.Repository, *scm.Response, error) {
+	path := fmt.Sprintf("search/repositories?%s", formRepoListQueryParams(opts))
+	out := new(searchRepositoryList)
+	res, err := s.client.do(ctx, "GET", path, nil, &out)
+	return convertSearchRepositoryList(out), res, err
```

Comment:
We can use the existing `convertRepositoryList` method here ? just pass `out.Repositories`

Reply:
done